.github/workflows/release.yml

```diff
@@ -55,6 +74,8 @@ env:
 
   BOX_CI_FOLDER_ID: 261814384
 
+  SIGNING_KEY: ${{ secrets.ONE_RELEASE_BUNDLES_GPG }}
```

Comment:
Why is needed if there's a input for the same purpose? Is the `input` the one that matters or this one?

Reply:
Removed ` SIGNING_KEY: ${{ secrets.ONE_RELEASE_BUNDLES_GPG }}` from the workflow .